bindings/jni/src/main/java/org/zeromq/czmq/Zconfig.java

```diff
@@ -238,10 +238,9 @@ public void removeSubtree () {
     }
     /*
     Destroy node and subtree (all children)
-    WARNING: manually fixed void -> long
```

Comment:
please revert changes to this file

Reply:
of course - although its generated by zproject automatically - so maybe we need to fix it there?

Reply:
yes - there's an outstanding issue, but nobody has provided a fix until now